_includes/landing-page/landing-demo.html

```diff
@@ -0,0 +1,68 @@
+<div class="container">
+    <div class="columns is-multiline">
+        <div class="column is-12 is-spaced-bottom">
+            <div class="columns">
+                <div class="column is-8 is-offset-2 has-text-centered">
+                    <h1 class="title is-size-3 is-size-2-desktop has-text-grey-darker">
+                        Simple, powerful functions from anywhere.
+                    </h1>
+                </div>
+            </div>
+        </div>
+        <div class="column is-12 has-text-centered  is-spaced-bottom">
+            <div class="columns">
+                <div class="column is-8 is-offset-2 has-text-centered">
+                    <h1 class="title has-text-weight-light is-size-5 is-size-4-desktop has-text-grey">
+                        Whether you want to use Go, Java, Python, C#, Ruby - Express.js,
+                        Django, ASP.NET Core or even binaries like ffmpeg,
+                        ImageMagick or anything else we have you covered.
+                    </h1>
+                </div>
+            </div>
+        </div>
+        <!-- To be activated -->
```

Comment:
This is great you added this in commented out so it'll be easier to implement once we're ready with the sample code. :100: